spec/features/1_basic_spec.rb

```diff
@@ -0,0 +1,175 @@
+require "rails_helper"
+
+describe "New User record" do
+  before do
+    sign_in_user if user_model_exists?
+  end
+  
+  it "has a default `likes_count` of 0", points: 1 do
+    expect(@user.likes_count).to eq(0),
+      "Expected a new user to have a default `likes_count` of 0. Did you make the change in your user migration file?"
+  end
+
+  it "has a default `comments_count` of 0", points: 1 do
+    expect(@user.comments_count).to eq(0),
+      "Expected a new user to have a default `comments_count` of 0. Did you make the change in your user migration file?"
+  end
+end
+
+describe "The home page" do
+  before do
+    sign_in_user if user_model_exists?
+  end
+
+  it "has a bootstrap navbar", points: 1 do
+    visit "/"
+
+    expect(page).to have_selector("nav[class^='navbar']"),
+      "Expected home page to have a bootstrap navbar <nav class='navbar ...'> ."
+  end
+
+  it "has an edit profile link for the signed in user", points: 1 do
+    visit "/"
+
+    expect(page).to have_selector("a[href='/users/edit']", text: "Edit #{@user.username}"),
+      "Expected home page to have 'Edit [USERNAME]' link with the username of the signed in user."
+  end
+
+  it "has a sign out link with a DELETE request for the signed in user", points: 1 do
+    visit "/"
+
+    expect(page).to have_selector("a[href='/users/sign_out'][data-method='delete']"),
```

Comment:
You could also check text here

```suggestion
    expect(page).to have_selector("a[href='/users/sign_out'][data-method='delete']", text: "Sign out"),
```

Reply:
Thanks, yeah.